templates/typescript-lambda-cdk/lib/lambda-ts-cdk-stack.ts

```diff
@@ -45,23 +38,46 @@ export class LambdaTsCdkStack extends cdk.Stack {
       },
     });
 
-    const environment = props.selfHosted
-      ? new restate.SingleNodeRestateDeployment(this, "Restate", {
-          logGroup: new logs.LogGroup(this, "RestateLogs", {
-            retention: logs.RetentionDays.THREE_MONTHS,
-            removalPolicy: cdk.RemovalPolicy.DESTROY,
-          }),
-        })
-      : new restate.RestateCloudEnvironment(this, "RestateCloud", {
-          clusterId: props.clusterId,
-          authTokenSecretArn: props.authTokenSecretArn,
-        });
+    const environment = new restate.SingleNodeRestateDeployment(this, "Restate", {
+      // restateImage: "docker.io/restatedev/restate",
+      // restateTag: "0.9",
```

Comment:
The construct defaults to `latest` but we can update the example to a concrete tag, once we release Restate. There is no hard dependency on the Restate version though; only the service deployer calls the `/deployments` API over HTTP.